web/css/src/utilities.css

```diff
@@ -200,3 +200,7 @@
 		display: none !important;
 	}
 }
+
+.js-password-length {
```

Comment:
`.js-` classes are typically reserved for use in JS, not CSS.

Reply:
Can this be an online style as it only is for the length of the "length" field, so that class will be removed?

Reply:
Yes 👍